### PR TITLE
To Do list: interactive list

### PR DESCRIPTION
**Pull request by rudicarrilloypr**
### Project requirements

- Add a new JavaScript file and import it as a module:    
    - it will contain methods related to the status updates (`completed`: `true` / `false`).
- Add event listener to the checkbox (`change`).
- Update items object's value for `completed` key upon user actions.
- Implement a function for the "Clear all completed" button (use `filter()` method).
- Store the updated array of items in local storage, so the user gets the correct list of values after the page reloads (which means that any changes made to the list should be preserved).

### CODE REVIEW

**Comment by rudicarrilloypr** [comment]
webhint linter is not working well, Microverse notified us to submit the project with that issue and code reviewers will ignore it, the rest of the linters and project requirements work properly

File: src/statusUpdates.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+// statusUpdates.js
+export function toggleTaskCompleted(task, populateTodoList, tasks, todoList) {
```
Comment (joskalenda) [inline src/statusUpdates.js:2]: - [x] This function is missing something, when I check ✔️  as task as completed, normally it should add a line through the task as well, but in your case, it does not do that, please fix it, take a look at [this](https://www.youtube.com/watch?v=AcUd-_Yjjqg) reference video for more.

YOURS
 


ORIGINAL

**Review by joskalenda — CHANGES_REQUESTED** [review]
| STATUS: REQUIRED CHANGES ♻️  |
|--------------------------------|
| Hi @rudicarrilloypr , |
|Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there! |
|  |

## Highlights

- [ ].. ✅ Clean code
- [ ].. ✅ Great design
- [ ].. ✅ Great readme file

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

-------
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@joskalenda) in your question so I can receive the notification.

> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/statusUpdates.js (hunk 1 above)
Comment (rudicarrilloypr) [reply src/statusUpdates.js:2]: @joskalenda but that is not required on the project requirements of this part of the project

**Review by Qoosim — APPROVED** [review]
#  *Status: Approved* :tada: :heavy_check_mark: 

Hi @rudicarrilloypr, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 
-  Good README file. :heavy_check_mark: 
-  Correct GitHub Flow. :heavy_check_mark: 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**